Update 1.9.3 changelog with MFA installation instructions (#237)
silverstripe/cwp: docs/en/05_Releases_and_changelogs/cwp_recipe_basic_1.9.3.md

To upgrade, adjust your projects composer.json to reflect the new constraints:

```json
    "require": {
        "cwp/cwp-recipe-basic": "^1.9.3@stable",
        "cwp/cwp-recipe-blog": "^1.9.3@stable",
        [...]
    },
```

## Multi-factor Authentication (MFA)

> In order to comply with CWP agreements in security auditing releases, this version has been released with a minor issue preventing MFA from operating in conjunction with the subsites module. Releases for both Framework and Subsites have been made since in order to rectify this, however as these were not included in the security audit for the 1.9.3 release agencies are encouraged to evaluate the decision to install these, and the impacts for them in deviating from the security audited CWP release.

Reviewer: ```suggestion
This version has been released with a minor issue preventing MFA from operating in conjunction with the subsites module. Releases for both Framework and Subsites have been made since in order to rectify this, however as these were not included in the security audit for the 1.9.3 release agencies are encouraged to evaluate the decision to install these, and the impacts for them in deviating from the security audited CWP release.
```

Reviewer: I don't think this sentence is required in the introduction.

> To install MFA on a CWP 1.9.3 based site, make the following amendments to the site's composer.json:

Reviewer: ```suggestion
In addition to following the installation steps of the [MFA module](https://github.com/silverstripe/silverstripe-mfa/tree/3.0), you will need to make the following amendments to the site's composer.json:
```

```json
    "require": {
        [...]
        "silverstripe/framework": "3.7.4 as 3.7.3",

>         "silverstripe/subsites": "1.4.2 as 1.4.1",

Reviewer: ```suggestion
        "silverstripe/subsites": "1.4.2 as 1.4.1"
```

>         "silverstripe/totp-authenticator": "3.0.0",

Reviewer: ```suggestion
```

>         "silverstripe/webauthn-authenticator": "3.0.0"

Reviewer: ```suggestion
```

Reviewer: These repos are only suggested. Sites should review the MFA repo and make this decision for themselves.
